src/Compilers/CSharp/Test/Emit3/Semantics/InlineArrayTests.cs

```diff
@@ -19491,8 +19491,73 @@ public T Current
     }
 }
 ";
+            var expectedOutput = " 111 112 113 114";
             var comp = CreateCompilation(src, targetFramework: TargetFramework.Net80, options: TestOptions.ReleaseExe);
-            CompileAndVerify(comp, expectedOutput: " 111 112 113 114").VerifyDiagnostics();
+            CompileAndVerify(comp, expectedOutput: expectedOutput).VerifyDiagnostics();
+
+            comp = CreateRuntimeAsyncCompilation(src);
+            // https://github.com/dotnet/roslyn/issues/79791: Verify runtime async output
```

Comment:
Should helper `ExpectedOutput(expectedOutput, isRuntimeAsync: true)` be used here like in other runtime async tests? (Same comment applies for a bunch of new tests after the 2nd commit.)